Products/ZenUI3/browser/resources/js/zenoss/Action.js

```diff
@@ -22,9 +21,6 @@
             if (!this.permitted) {
                 return true;
             }
-            if (this.externallyDisabled) {
-                return true;
-            }
             if (!this.permitted && !this.filtered) {
```

Comment:
This `if` statement will never succeed or never execute because the `if` statement on line 21 will return when `this.permitted` is false.